README.md

```diff
@@ -72,6 +72,21 @@ server is running
 to the IP of your Clio server. This entry can take the form of a comma-separated list if
 you are running multiple Clio nodes.
 
+In addition, the parameter `start_sequence` can be included and configured within the top level of the config file. This parameter specifies the sequence of first ledger to extract if the database is empty. Note that ETL extracts ledgers in order and that no backfilling functionality currently exists, meaning Clio will not retroactively learn ledgers older than the one you specify. Choosing to specify this or not will yield the following behavior:
+- If this setting is absent and the database is empty, ETL will start with the next ledger validated by the network. 
+- If this setting is present and the database is not empty, an exception is thrown.
+
+In addition, the optional parameter `finish_sequence` can be added to the json file as well, specifying where the ledger can stop.
+
+To add `start_sequence` and/or `finish_sequence` to the config.json file appropriately, they will be on the same top level of precedence as other parameters (such as database, etl_sources, read_only, etc.) and be specified with an integer. Here is an example below, added at the end of the config.json:
+
+```sh
```

Comment:
Is there no `json` variant for syntax highlighting?

Reply:
@godexsoft not that I'm aware of for plaintext, but there is for code if encapsulated in ```sh ```to give the highlighting you would find on a CLI
![image](https://user-images.githubusercontent.com/47958858/207117019-57bcc201-e220-4eb9-b1c8-4710c8c5221a.png)

Reply:
Please try both `json` and then `javascript` and see if anything looks like actual json and not `shell` after syntax highlighting on github.